concepts/functions/about.md

```diff
@@ -0,0 +1,153 @@
+  That means you are dealing with a copy of the original value in the function body and you can modify it without affecting the original value.
+- All other values (objects, arrays, functions) are _passed by reference_.
+  If you modify arguments of non-primitive types, you are changing the original value outside of the function because the argument represents a reference to the original value, not a copy of that value.
+
+By default, all parameters defined in the function declaration are optional in JavaScript.
+If you provide less arguments than there are parameters, the missing arguments will be `undefined` inside the function, see [Null and Undefined][concept-null-undefined].
+In many cases it makes sense to assign a more appropriate default value than `undefined`. This can by done by specifying default parameters directly in the function definition.
+
+```javascript
+function someName(param1 = defaultValue1, param2 = defaultValue2) {
+  // ...
+}
+```
+
+You can even call a function with more arguments than there were parameters in the definition.
+Then the excess arguments are present in the [arguments "array"][mdn-arguments-object].
+
+It is also possible to define functions that accept an arbitrary number of arguments (variadic functions), see rest parameters in [Rest and Spread Operators][concept-rest-and-spread] for details about this.
+
+## Return Statement
+
+Using the `return` statement, you can pass the result of a function to code that called it.
+There can be multiple `return` statements in a function.
+The execution of the function ends as soon as it hits one of those `return`s.
+
+```javascript
+function checkNumber(num) {
+  if (num === 0) {
+    return 'You passed 0, please provide another number.';
+  }
+
+  return 'Thanks for passing such a nice number.';
+}
+```
+
+If you use a naked return or no return at all, the result of the function is `undefined`.
+There are no implicit returns in JavaScript.
+
+```javascript
+function nakedReturn(a) {
+  a * 2;
+  return;
+}
+
+nakedReturn(1);
+// => undefined
+
+function noReturn(a) {
+  a * 2;
+}
+
+noReturn(1);
+// => undefined
+```
+
+In JavaScript, you can only return exactly one value.
+If you want to pass more information, you need to combine it into one entity first, usually into an [object][concept-objects].
+
+```javascript
+function divide(a, b) {
+  return {
+    quotient: Math.floor(a / b),
+    remainder: a % b,
+  };
+}
+```
+
+## Function Expression
+
+A function declaration is a standalone statement.
+But sometimes it is helpful to define a function as part of another expression, e.g., in an assignment, as a function parameter ([callback][concept-callbacks]) or as value in an [object][concept-objects].
+This can be done with a function expression.
+It has the same syntax as a function declaration, only that the function name can be omitted to create an _anonymous function_.
```

Comment:
We don't need to mention this now, as this about.md will evolve later, as we discussed, but it _might_ be interesting to note that whilst all three are examples of _anonymous function expressions_, only one of them is results in an _anonymous function_.

Reply:
I added your comment to the list in the follow up ticket.